config/mqtt-client.php

```diff
@@ -55,6 +55,9 @@
             // with the log level as configured.
             'enable_logging' => env('MQTT_ENABLE_LOGGING', true),
 
+            // Which logging channel to use or to broadcast to all available
```

Comment:
The logs are not broadcasted but the default log channel is used (may be a stack):
```suggestion
            // Which logging channel to use for logs produced by the MQTT client.
            // If left empty, the default log channel or stack is being used.
```